sql/analyze.go

```diff
@@ -1222,8 +1222,7 @@ func simplifyComparisonExpr(n *parser.ComparisonExpr) parser.Expr {
 			return n
 		case parser.In, parser.NotIn:
 			tuple := n.Right.(parser.DTuple)
-			sort.Sort(tuple)
-			tuple = uniqTuple(tuple)
+			tuple.Normalize()
```

Comment:
Ah right, I didn't see that `scanNode.initWhere` was already normalizing the `Where` clause for index selection before `analyzeExpr` is called in `planner.Select`. Removed.